test/common-utils/scenarios.json

```diff
@@ -0,0 +1,10 @@
+{
+  "configure_zsh_as_default_shell": {
+    "image": "ubuntu:focal",
```

Comment:
```suggestion
    "image": "mcr.microsoft.com/devcontainers/base:ubuntu",
```

💡  If you'd change the `image`, then in the test instead of hardcoding `vscode`, you could use `$(whoami)`